misk-admin/api/misk-admin.api

```diff
@@ -218,7 +218,12 @@ public final class misk/web/dashboard/DashboardHomeUrl : misk/web/dashboard/Vali
 }
 
 public final class misk/web/dashboard/DashboardModule : misk/inject/KAbstractModule {
```

Comment:
The existing `DashboardModule` wasn't used anywhere outside of Misk so this is backwards compatible.